### PR TITLE
Home page mobile version

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/components/utility/hStack/HStack.tsx [hunk 1]
```diff
@@ -7,7 +8,7 @@ export interface HStackProps {
 
 export default function HStack(props: React.PropsWithChildren<HStackProps>): ReactElement {
     return (
-        <span className={props.className} style={{ display: 'flex', flexDirection: 'row', justifyContent: 'center', ...props.style }}>
+        <span className={classNames('flex', props.className)} style={{ flexDirection: 'row', justifyContent: 'center', ...props.style }}>
```
Comment (CemYil03) [inline src/components/utility/hStack/HStack.tsx:11]: I would like to keep these utils free from dependencies, would it be okay if we leave them as they were?

File: src/components/standard/bulletPoint/PEBulletPoint.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { type ReactElement } from 'react';
+import VStack from '~/components/utility/vStack/VStack';
 import HStack from '../../utility/hStack/HStack';
```
Comment (CemYil03) [inline src/components/standard/bulletPoint/PEBulletPoint.tsx:3]: Tilde import

```suggestion
import HStack from '../../utility/hStack/HStack';
import VStack from '../../utility/vStack/VStack';
```

File: src/components/pages/home/index.tsx [hunk 3]
```diff
@@ -41,8 +46,6 @@ export default function HomePage(): ReactElement {
     const [date, setDate] = useState(moment());
     const [searchResults, setSearchResults] = useState<GoogleMapsPlacesResult[]>([]);
 
-    const [selectedLocation, setSelectedLocation] = useState<{ latitude: number; longitude: number } | undefined>(undefined);
-
```
Comment (CemYil03) [inline src/components/pages/home/index.tsx:45]: I think you can't delete them. This is the reason the TypeScript build fails now

File: src/components/cards/pointsCard/PEPointsCard.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            </HStack>
+            <PEButton
+                className={'mt-8 sm:max-w-full max-w-[270px] bg-transparent'}
+                type={'secondary'}
+                onClick={(): void => undefined}
+                title={'Send an individual request'}
+            />
+        </div>
+    );
+}
```
Comment (CemYil03) [inline src/components/cards/pointsCard/PEPointsCard.tsx:39]: Are you sure this makes sense? Now the single points are still not a component...

File: src/components/cards/chefCard/PEChefCard.tsx [hunk 5]
```diff
@@ -58,14 +58,17 @@ export default function PEChefCard({
                         </div>
                     </div>
                 </div>
-                <div className={'no-scrollbar overflow-x-scroll flex flex-row gap-2 mt-3 scrollbar-hide'}>
+                <div className={'no-scrollbar overflow-x-scroll flex flex-row gap-2 mt-3 scrollbar-hide'} style={{ overflowY: 'initial' }}>
```
Comment (CemYil03) [inline src/components/cards/chefCard/PEChefCard.tsx:61]: What is the `overflowY: 'initial'` there for?

File: tailwind.config.cjs [hunk 6]
```diff
@@ -89,6 +91,29 @@ const config = {
...
+
+                sm_max: { min: '640px' },
+
+                sm: { max: '639px' },
+
+                sm_min: { min: '518px', max: '639px' },
+
+                minn: { min: '318px', max: '518px' },
+                // => @media (max-width: 639px) { ... }
+            },
```
Comment (CemYil03) [inline tailwind.config.cjs:116]: Does tailwind not have defaults for this?

File: src/components/pages/home/search/HomePageSearchMobile.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+                            onAddressSearchTextChange(event.target.value);
+                        }}
+                        variant="standard"
+                        label={t('search-city-label')}
+                        InputProps={{ disableUnderline: true, ...params.InputProps }}
+                        InputLabelProps={{ shrink: true }}
+                    />
+                )}
+            />
+            <PEIconButton bg={'bg-orange'} icon={Icon.filters} size={'40px'} iconSize={24} className={'rounded-4'} />
```
Comment (CemYil03) [inline src/components/pages/home/search/HomePageSearchMobile.tsx:56]:
```suggestion
            <PEIconButton bg="bg-orange" icon={Icon.filters} size={'40px'} iconSize={24} className="rounded-4"
            />
```

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/cards/chefCard/PEChefCard.tsx (hunk 5 above)
Comment (Llavrov) [reply src/components/cards/chefCard/PEChefCard.tsx:61]: to switch off scroll behavior for Y ordinate

File: tailwind.config.cjs (hunk 6 above)
Comment (Llavrov) [reply tailwind.config.cjs:116]: Tried to use defaults variables, but it didn't work for me(

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**